### ISSUE TITLE
Feature Request - Remote Screen Draw feature

### ISSUE THREAD

**Issue by rogalskij13** [Ylianst/MeshCentral#1639]
Not sure if this is the right place to request, but would it be possible to add a remote screen drawing utility?  So often staff have trouble seeing the thing I am requesting them to click on to type into.  This would be a method where I could walk a user through something and say "please double click this button on screen" while I could draw a circle around the object in question.  They would see the circle and know exactly where I am asking them to click or type.  

This is common in some other screen sharing software I have used previously.  Thank you for your consideration!

**Comment by Ylianst** [comment]
Yes, many other remote desktop software have this. Certainly a valid request.
